django/db/backends/base/schema.py

```diff
@@ -1063,9 +1063,20 @@ def _field_should_be_altered(self, old_field, new_field):
         _, new_path, new_args, new_kwargs = new_field.deconstruct()
         # Don't alter when:
         # - changing only a field name
+        # - changing an attribute that doesn't affect the schema
         # - adding only a db_column and the column name is not changed
-        old_kwargs.pop('db_column', None)
-        new_kwargs.pop('db_column', None)
+        fields_that_do_not_trigger_a_schema_change = [
```

Comment:
Perhaps a better, shorter name would be:

```suggestion
        non_database_fields = [
```

Reply:
Thanks! Way better then what I came up with 💯